stock_picking_report_delivery_cost/report/report_shipping.xml

```diff
@@ -6,10 +6,10 @@
         <xpath expr="//div[@name='div_sched_date']" position="after">
             <div
                 t-if="o.carrier_price_for_report"
-                class="col-auto"
+                class="col"
                 name="div_carrier_price_for_report"
             >
-                <strong>Delivery cost:</strong>
+                <strong>Delivery cost</strong>
```

Comment:
This change is making all the current translations invalid. Is it needed? If so, please adapt all the translated PO files to remove the `:` both in the source and the translation.